test/cases/ostree-ng.sh

```diff
@@ -297,6 +297,14 @@ build_image container "${CONTAINER_TYPE}"
 greenprint "📥 Downloading the container image"
 sudo composer-cli compose image "${COMPOSE_ID}" > /dev/null
 
+# Save the build manifest and logs to the artifacts directory
+greenprint "Saving build metadata and logs"
```

Comment:
@achilleas-k We have fetch build log and metadata on function `get_compose_log` and `get_compose_metadata` already. But we need archive them.

Reply:
Perfect thank you.  I've updated the commit to use those functions.